assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -253,6 +285,42 @@ def _update_conda_dependencies(self, package_details):
             yaml.safe_dump(conda_dict, f)
             logger.info("updated conda.yaml")
 
+    def _get_curated_environment_pip_package_list(self) -> List[str]:
```

Comment:
Could you please share context on this?

Reply:
The base environment is using a pkg called apex, whose version is not in pypi, hence when we let mlflow create the conda file from the base env, it picks up apex for certain models and fails the local validation as it is unable to find the package in pypi. We know exactly what packages are needed and which ones need to be pinned for version, this method creates the conda yaml file that we require and that is sent to mlflow.transformers.save_model()